.env.sample

```diff
@@ -0,0 +1 @@
+SAMPLE_VARIABLE=
```

Comment:
should this file be committed?

Reply:
Yes, it's a sample env file that will have the variables but not the values